src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Resources/Resources.Designer.cs

```diff
@@ -52,55 +52,64 @@ public class Resources {
         ///   resource lookups using this strongly typed resource class.
         /// </summary>
         [global::System.ComponentModel.EditorBrowsableAttribute(global::System.ComponentModel.EditorBrowsableState.Advanced)]
-        public static global::System.Globalization.CultureInfo Culture {
+        internal static global::System.Globalization.CultureInfo Culture {
             get {
                 return resourceCulture;
             }
             set {
                 resourceCulture = value;
             }
         }
-
+        
+        /// <summary>
+        ///   Looks up a localized string similar to The active Multi Test Runs Finalization was aborted..
+        /// </summary>
+        internal static string AbortedMultiTestRunFinalization {
+            get {
+                return ResourceManager.GetString("AbortedMultiTestRunFinalization", resourceCulture);
+            }
+        }
+        
         /// <summary>
         ///   Looks up a localized string similar to The active Tests Discovery was aborted..
         /// </summary>
-        public static string AbortedTestsDiscovery {
+        internal static string AbortedTestsDiscovery {
             get {
                 return ResourceManager.GetString("AbortedTestsDiscovery", resourceCulture);
             }
         }
-
+        
         /// <summary>
         ///   Looks up a localized string similar to The active Tests Run was aborted..
         /// </summary>
-        public static string AbortedTestsRun {
+        internal static string AbortedTestsRun {
```

Comment:
>internal [](start = 8, length = 8)

Was this intentionally made internal? I'm not sure if this was meant to be used by external folks.

Reply:
this file is automatically generated. All other stuff is also internal

Reply:
Oh the access modifiers for a resource file can be updated in VS. It's the strip just above all the strings. Looks like for some reason that got updated to internal in this PR (perhaps adding a new string causes that).
![image](https://user-images.githubusercontent.com/18014426/85186183-78abb900-b24c-11ea-9694-fbba8f46a059.png)

Reply:
fixed, it looks like wrong value was inside csproj